032-MLOpsFromScratch/Coach/00-prereqs.md

```diff
@@ -0,0 +1,36 @@
+# Challenge 0: Pre-requisites
```

Comment:
First thing is first.... you need to have a README.md in the root of the Coach folder. This is the table of contents for the Coaches Guide.

For an example look in 001 hack:
https://github.com/microsoft/WhatTheHack/blob/master/001-IntroToKubernetes/Coach/README.md

Reply:
Added a README.md file in the Coach folder.

Reply:
Added a Readme file in the Coach folder.